### PR TITLE
Expose logged-in state to ghosterysearch page via content-script.

### PR DESCRIPTION
**Pull request by sammacbeth**
Instead of sending flags to the search endpoint, add classes to `html` in content script so we can toggle features with CSS.

### CODE REVIEW

File: src/content/is-ghostery-browser.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"use strict";
+
+(async function () {
```
Comment (chrmod) [inline src/content/is-ghostery-browser.js]: why async? does it need one tick? maybe requestAnimationFrame would be faster?

File: src/status.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+`;
+
+function injectLoggedInStatus(status) {
+  statusContentScript = statusContentScript.then(async () => {
+    if (status === loggedIn) {
+      return;
+    }
+    loggedIn = status;
+    // register a content script to inject status classes in the future
+    await browser.contentScripts.register({
```
Comment (chrmod) [inline src/status.js]: will this replace the previously registered content script?

File: src/status.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    const activeTabs = await browser.tabs.query({ url: `${SERP_BASE_URL}/*` });
+    activeTabs.forEach(({ id }) => {
+      browser.tabs.executeScript(id, {
+        code: status ? loggedInScript : loggedOutScript,
+      });
+    });
+  });
+}
+
+injectLoggedInStatus(false);
```
Comment (chrmod) [inline src/status.js]: Not sure if there is a need for doing that.

File: src/status.js (hunk 2 above)
Comment (sammacbeth) [reply src/status.js]: No, I forgot the deregistration part...

File: src/status.js (hunk 3 above)
Comment (sammacbeth) [reply src/status.js]: Removed

File: src/content/is-ghostery-browser.js (hunk 1 above)
Comment (sammacbeth) [reply src/content/is-ghostery-browser.js]: async was by mistake. Removed.

**Review by chrmod — APPROVED** [review]

**Pull request merged**